features/tabelwaarden.feature

```diff
@@ -47,6 +47,7 @@ De onderstaanden tabellen met de beschreven waarden worden geleverd door de "BRP
     | M                                                | ministerieel besluit                 |
     | R                                                | persoonslijst is aangelegd in de rni |
     | F                                                | fout                                 |
+    | .                                                | onbekend                             |
```

Comment:
Moet deze wijziging dan ook niet voor soort_verbintenis worden gedaan ?

Reply:
> Moet deze wijziging dan ook niet voor soort_verbintenis worden gedaan ?

Ik denk het niet. Wat is het verschil tussen geen soort verbintenis en een onbekende soort verbintenis?

Ik denk dat de volledig onbekende partner (dus alles, inclusief soort verbintenis is onbekend) op een andere manier moet worden opgelost.